webui/src/data/server.js

```diff
@@ -510,14 +510,17 @@ class ServerCache {
             ajaxGet({
                 url: apiUrls.languages(),
                 successFn: (data) => {
-                    const langs = data.languages.map(([id, name]) => ({id, name}));
+                    const langs = data.languages.map(([id,name]) => ({"id": id, "name": name}));
```

Comment:
The original is perfectly valid, see here: https://github.com/lukehoban/es6features#enhanced-object-literals